docs/registry/service-binding-registry/README.adoc

```diff
@@ -0,0 +1,597 @@
+
+.. If you connected to a {registry} instance, use Role-Based Access Control (RBAC) to enable the new service account to access the {registry} instance and the artifacts (such as schemas) that it contains.
++
+.Setting {registry} access permissions for the service account
+[source,options="nowrap",subs="+quotes"]
+----
+rhoas service-registry role add --role=manager --service-account __<my-client-id>__
+Updating role for principal
+Role was successfully applied
+----
++
+In the preceding command, replace __<my-client-id>__ with the `Client ID` value for the new service account. This value was previously shown on the command line.
++
+In this example, the `manager` role that you assign to the service account allows applications to use the service account to view and write to schemas in the {registry} instance.
+
+. Use the OpenShift CLI to verify that the RHOAS Operator successfully created the `KafkaConnection` or `ServiceRegistryConnection` object, as shown in the following example:
++
+.Using the OpenShift CLI to verify Operator connection to your cluster
+[source]
+----
+$ oc get KafkaConnection
+
+NAME   		         AGE
+my-kafka-instance    2m35s
+----
++
+As indicated by this output, when you use the `rhoas cluster connect` command, the RHOAS Operator creates a `KafkaConnection` or `ServiceRegistryConnection` object that matches the name of your Kafka or {registry} instance. In the preceding example, the object name matches a Kafka instance called `my-kafka-instance`.
+
+. Repeat the preceding steps to ensure that *both* your {product-kafka} and {registry} instances are connected to your OpenShift cluster.
+
+[id="con-kafka-registry-binding-quarkus-application-using-cli_{context}"]
+== Binding a Quarkus application to {product-long-kafka} and {product-long-registry} using the RHOAS CLI
+
+When the RHOAS Operator and Service Binding Operator are installed on your OpenShift cluster and you've connected a Kafka and {registry} instance to the cluster, you're ready to deploy an application and perform service binding. __Service binding__ means instructing the Service Binding Operator to automatically inject the application with the parameters required to connect to the Kafka and {registry} instances.
+
+The following tutorial shows how to use the RHOAS CLI to perform service binding. In the tutorial, you create an example Quarkus application and connect this to a Kafka and {registry} instance. link:https://quarkus.io/[Quarkus^] is a Kubernetes-native Java framework that is optimized for serverless, cloud, and Kubernetes environments. The Quarkus application in the tutorial uses an Apache Avro schema to serialize and deserialize messages. The application automatically publishes the Avro schema that it uses to your {registry} instance.
```

Comment:
@EricWittmann I expanded this paragraph a bit since the last review. Can you verify what I've said here?

Reply:
Looks good